sphinx_intl/commands.pyi

```diff
@@ -0,0 +1,67 @@
+# Type annotations for commands.py
+
+from typing import Text, Any, Mapping, Tuple, List, Sequence, Callable, Union
+
+
+import click
+
+ENVVAR_PREFIX: Text
+
+
+def main(ctx: Any,
+         config: Any,
+         tag: Any
+         ) -> None: ...
+
+def update(locale_dir: Text,
+           pot_dir: Text,
+           language: Text
+           ) -> None: ...
+
+def build(locale_dir: Text,
+           pot_dir: Text,
+           language: Text
+          ) -> None: ...
+
+def stat(locale_dir: Text,
+         language: Text
+         ) -> None: ...
+
+def create_transifexrc(transifex_username: Text,
+                       transifex_password: Text
+                       ) -> None: ...
+
+def create_txconfig() -> None: ...
+
+def update_txconfig_resources(transifex_project_name: Text,
+                              locale_dir: Text,
+                              pot_dir: Text
+                              ) -> None: ...
+
+def read_config(path: Text,
+                passed_tags: Sequence[Text]
+                ) -> Mapping[Text, Text]: ...
+
+def get_lang_dirs(path: Text) -> Tuple[Tuple[List[Text]]]: ...
+
+
+option_locale_dir: Callable
+option_pot_dir: Callable
+option_output_dir: Callable
+option_tag: Callable
+option_language: Callable
+option_transifex_username: Callable
+option_transifex_password: Callable
+option_transifex_project_name: Callable
+CONTEXT_SETTINGS: Mapping[Text, Sequence[Text]]
+
+
+class LanguagesType(click.ParamType):
+    name: Union[Text, unicode]= ...
+    envvar_list_splitter: Union[Text, unicode]= ...
```

Comment:
Unfortunately, this expression is interpreted as `Union[unicode, unicode]`.

So I've changed these 2 lines in my local as just `str`, mypy testing has been passed.

Reply:
In order to execute mypy in CI test, please add below part in `setup.cfg` and `tox.ini` that copied from sphinx project.

`setup.cfg`:
```
[mypy]
python_version = 2.7
show_column_numbers = True
show_error_context = True
ignore_missing_imports = True
follow_imports = skip
incremental = True
check_untyped_defs = True
warn_unused_ignores = True
strict_optional = False
```

tox.ini:
```
envlist=py36,py35,py34,py27,flake8,sdist,mypy

...

[testenv:mypy]
basepython = python3
description =
    Run type checks.
deps =
    mypy
commands=
    mypy sphinx_intl/
```